WordPress/src/main/res/layout/fragment_login_no_sites_empty_view.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_extra_large"
+        android:gravity="center|center_horizontal|center_vertical">
+
+        <androidx.constraintlayout.widget.Group
+            android:id="@+id/login_empty_view_user_section"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:constraint_referenced_ids="image_avatar,text_username, text_displayname"
+            tools:visibility="visible"/>
+
+        <ImageView
+            android:id="@+id/image_avatar"
+            android:layout_width="@dimen/login_empty_view_avatar_width"
+            android:layout_height="@dimen/login_empty_view_avatar_height"
+            android:layout_marginBottom="@dimen/login_empty_view_avatar_margin_bottom"
```

Comment:
Question (❓): The design specifies that this is `16dp` instead. Am I missing something?

Reply:
@ParaskP7 Same comment as below on specs